### PR TITLE
fix: make toggleTermsTickbox for pro wallet mandatory

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: rockwallet/rockwallet/src/Redesign/Scenes/SignUp/SignUpInteractor.swift [hunk 1]
```diff
@@ -71,8 +71,14 @@ class SignUpInteractor: NSObject, Interactor, SignUpViewActions {
     }
     
     func toggleTermsTickbox(viewAction: SignUpModels.TermsTickbox.ViewAction) {
-        dataStore?.termsTickbox = viewAction.value
+        guard let valuePro = viewAction.valuePro else {
```
Comment (kenanmamedoff) [inline rockwallet/rockwallet/src/Redesign/Scenes/SignUp/SignUpInteractor.swift:74]: can't you do 

`dataStore?.termsTickbox = viewAction.value ?? viewAction.valuePro`

Comment (dangelovska) [reply rockwallet/rockwallet/src/Redesign/Scenes/SignUp/SignUpInteractor.swift:74]: I have two different variables in data store **dataStore?.termsTickbox** and **dataStore?.termsTickboxPro** to store the values

Comment (dangelovska) [reply rockwallet/rockwallet/src/Redesign/Scenes/SignUp/SignUpInteractor.swift:74]: both values ​​need to be true so we can proceed with registration

Comment (kenanmamedoff) [reply rockwallet/rockwallet/src/Redesign/Scenes/SignUp/SignUpInteractor.swift:74]: if else would be better then. no need to have a guard. this is not really readable

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**